vocabularies/Common.json

```diff
@@ -1288,6 +1288,132 @@
       "@Common.Experimental": true,
       "@Core.IsURL": true,
       "@Core.Description": "Base URL for WebSocket connections"
+    },
+    "RecommendationsFunction": {
+      "$Kind": "Term",
+      "$Type": "Common.QualifiedName",
+      "$AppliesTo": ["EntityType"],
+      "@Common.Experimental": true,
+      "@Core.Description": "Qualified name of a function that computes AI-based recommendations for entities of the annotated type (see [this template](#Template_RecommendationsFunction) and [this alternative](#Template_RecommendationsFunction2))"
+    },
+    "Template_RecommendationsFunction": [
+      {
+        "$Kind": "Function",
```

Comment:
Probably discussed elsewhere already: Is this one really a Function? It creates some state on the server that can be read at a different (!) URL (the returned one)

Reply:
But the state is not really persistent on the server. It is deleted after it has been read (or when it is older than one minute or so). And it can only be accessed by the returned URL, so it is invisible to everyone else.

Reply:
It doesn't change the "business" data of its defining service, which is good enough for me.

Most/all HTTP calls create state somewhere, for example in access logs.

Reply:
thanks! ✅